modin/core/storage_formats/pandas/parsers.py

```diff
@@ -227,6 +230,8 @@ def get_dtypes(cls, dtypes_ids):
             index and types of columns are used as values for full resulting
             frame.
         """
+        if len(dtypes_ids) == 0:
+            return None
```

Comment:
shouldn't we just return an empty series?

Reply:
This value is then passed to `PandasDataframe` inside `dtypes` parameter which default values is None. So I think we shouldn't.

Reply:
I will update the docstring.